pkg/cmd/run.go

```diff
@@ -105,12 +105,13 @@ func RunMicroshift(cfg *config.MicroshiftConfig, flags *pflag.FlagSet) error {
 				if err := node.StartKubelet(cfg); err != nil {
 					return err
 				}
-				if err := node.StartKubeProxy(cfg); err != nil {
-					return err
-				}
+				// if err := node.StartKubeProxy(cfg); err != nil {
+				// 	return err
+				// }
 				return nil
 			},
 		)))
+		util.Must(m.AddService(kustomize.NewKustomizer(cfg)))
```

Comment:
Why is kustomizer service moved from "only control plane" to all kinds of hosts?

Reply:
Must have been a typing error. Will fix it.